test/unit/config.spec.js

```diff
@@ -30,6 +48,21 @@ describe('config', () => {
   const wrapCfg = function (cfg) {
     return (config) => config.set(cfg)
   }
+  function wrapCfgResolvedPromise (cfg) {
```

Comment:
No we don't.

<details>

I thought it was going to be used more than once like the other wrapper. It could be inlined like some of the other mocks. That said, it is also a reasonable enough wrapper with the potential to see more use if more async mocks are needed in the future.

</details>

At the very least I will change it to match your suggestion. I'm fine with moving it directly to the mock's property as well.